README.md

```diff
@@ -70,6 +70,8 @@ After cloning Juvix into a local directory, go into the local Juvix directory, a
 ```bash
 make
 ```
+Expect the installation to take some time (~ 40 minutes).  
```

Comment:
the time will depend on the machine's specs. Maybe mention you machine's spec? 
```suggestion
Expect the installation to take some time (~ 40 minutes on a ?? machine).  
```

Reply:
ok so let's not give any time indication, but just mention that it might take some time.